Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM centos:8
+
+ENV ansible_dir=/root/JetSki/ansible-ipi-install
+ENV DISABLE_PODMAN=true
+
+RUN yum -y install python3 --nodocs
+RUN pip3 install ansible
+RUN yum -y install epel-release --nodocs
+RUN yum --enablerepo=epel -y install sshpass
+RUN yum -y install openssh-clients --nodocs
+RUN yum -y install git --nodocs
+RUN pip3 install jmespath
```

Comment:
What do we need this for?

Reply:
Several repositories here use epel.

openssh-clients is needed for at least one task run by the playbook. It cannot connect to a remote server over SSH without it.
Git is needed to clone the repository.
jmespath is another requirement of at least one task run by the playbook.

Reply:
Yeh I was curious about jmespath not the others..cool